v1.go

```diff
@@ -54,28 +54,20 @@ func v1MountPoint() (string, error) {
 	defer f.Close()
 	scanner := bufio.NewScanner(f)
 	for scanner.Scan() {
-		if err := scanner.Err(); err != nil {
-			return "", err
-		}
 		var (
-			text   = scanner.Text()
-			fields = strings.Split(text, " ")
-			// safe as mountinfo encodes mountpoints with spaces as \040.
-			index               = strings.Index(text, " - ")
-			postSeparatorFields = strings.Fields(text[index+3:])
-			numPostFields       = len(postSeparatorFields)
+			text      = scanner.Text()
+			fields    = strings.Split(text, " ")
+			numFields = len(fields)
 		)
-		// this is an error as we can't detect if the mount is for "cgroup"
-		if numPostFields == 0 {
-			return "", fmt.Errorf("Found no fields post '-' in %q", text)
+		if numFields < 10 {
```

Comment:
i wonder if this is going to break anything, i remember some kernels have in the best, returned various numbers in the mount outputs

Reply:
I am aware of two kernel bugs:

1. empty "source" field. this is not affecting this parser because it uses strings.Split by a space, and the extra space is there (for details, see https://github.com/moby/sys/pull/5).

2. extra fields added by cifs mounts with `unc=` option with value containing spaces. This makes numFileds greater, not less.

IOW I am not aware of anything that would break this condition.

Reply:
ok, thanks.  just wanted to double check